apps/i18n/common/en_us.json

```diff
@@ -329,13 +329,33 @@
   "defaultProjectName": "My Project",
   "delete": "Delete",
   "deleteAccount": "Delete Account",
-  "deleteAccount_warning": "Deleting your account will permanently erase all personal information, coursework, and projects connected to this account.",
+  "deleteAccount_studentWarning": "Deleting your account will permanently erase all personal information, coursework, and projects connected to this account.",
+  "deleteAccount_teacherWarning1": "Deleting your account will permanently erase all personal information, coursework, and projects connected to this account after 28 days. It will also ",
+  "deleteAccount_teacherWarning2": "delete your sections and your students’ accounts ",
+  "deleteAccount_teacherWarning3": "that don’t have a personal login or aren’t in another teacher’s section. Please make sure you have the authority to delete these students’ education records before deleting your own account. ",
+  "deleteAccount_teacherWarning4": "Give these students a chance to keep using their Code.org accounts by ",
+  "deleteAccount_teacherWarning5": "sending home instructions for creating personal logins. ",
+  "deleteAccount_teacherWarning6": "Give them at least a few days to follow these instructions BEFORE you delete your account.",
```

Comment:
@Hamms Sorry to bug you, but what would it take for us to get redacted markdown working for apps translations like this?  We've been seeing this more and more in accounts/teacher dashboard tools, and are really looking forward to markdown support.

Reply:
No worries! To clarify, do we actually need _redacted_ markdown support or just markdown support?

Reply:
i think both in this case. most of the text just requires styling, but there are a few that are links that would need to be redacted

Reply:
Translated markdown, which I assumed (for security reasons) meant we needed redacted markdown.

Reply:
Ah, looking closer at what you're doing here, it seems like the answer is "both".

I would, however, say they are two separate issues. Implementing redacted markdown will make sending stuff that contains things like URLs or other immutable/sensitive information off to translators safe and easy, but we will still need a way to _render_ that markdown in an apps context